src/main/java/org/embulk/input/gcs/GcsFileInputPlugin.java

```diff
@@ -446,7 +454,16 @@ public InputStream openNext() throws IOException
             try (BufferedOutputStream outputStream = new BufferedOutputStream(new FileOutputStream(tempFile))) {
                 IOUtils.copy(getObject.executeMediaAsInputStream(), outputStream);
             }
-            return new ResumableInputStream(new BufferedInputStream(new FileInputStream(tempFile)), new GcsInputStreamReopener(tempFile, client, bucket, key, maxConnectionRetry));
+            catch (Exception ex) {
+                return new ResumableInputStream(
+                    new BufferedInputStream(new FileInputStream(tempFile)),
+                    new GcsInputStreamReopener(tempFile, client, bucket, key, maxConnectionRetry, ex)
+                );
```

Comment:
Can you explain why this works? When `IOUtils.copy` method throws a certain exception like `IOException`, the temp file sometimes will be broken. In that case, we'd better not to create `ResumableInputStream` object?